configs/nextflow.hopper.config

```diff
@@ -79,6 +80,9 @@ params {
 
   // SLURM MISC //
   queue             = 'normal'
+
+  // CONTAINERS //
+  container_fastp = '/fs1/resources/containers/fastp_0.23.4.sif'
```

Comment:
`sentieon_container` vs `container_fastp`. 

Nitpick but we should remember to harmonize those vars when prepping for release.

Reply:
Yes

Reply:
Renamed to `container_sentieon`